lib/js/test/tests/nft_forwarding_test.test.js

```diff
@@ -145,12 +145,12 @@ describe("NFTForwarding Contract Tests", () => {
         await setupForwarding(forwarderAccount, recipientAccount);
 
         // Unlink collection in recipientAccount
-        const [unlinkTxn, e1] = await shallPass(
+        await shallPass(
             sendTransaction("unlink_collection", [recipientAccount], [ ])
         );
 
         // Transfer NFT, but attempt to forward should fail
-        const [transferTxn, e2] = await shallRevert(
+        await shallRevert(
```

Comment:
I prefer to make sure the revert string is not null as there are some cases where revert can happen because of some unexpected things in code execution. The current way it is not making sure that it is failing at the same desired statement.

Reply:
any input here @sisyphusSmiling?

Reply:
I agree. Submitted changes in the recent commits